prompts/large.md

```
@@ -0,0 +1,500 @@
+
+```bash
+parseArger project my-awesome-project \
+  --description "This is a cool project" \
+  --git-repo "user/my-awesome-project" \
+  --project-subcommand build \
+  --project-subcommand deploy \
+  --project-subcommand test
+```
+
+**Creates structure:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)